pantheon/src/main/java/tech/pegasys/pantheon/controller/CliquePantheonControllerBuilder.java

```diff
@@ -121,6 +126,22 @@ protected MiningCoordinator createMiningCoordinator(
         genesisConfig.getConfigOptions(), nodeKeys, privacyParameters);
   }
 
+  @Override
+  protected boolean validateContext(final ProtocolContext<CliqueContext> context) {
+    final Optional<BlockHeader> genesisBlockHeader =
+        context.getBlockchain().getBlockHeader(GENESIS_BLOCK_NUMBER);
+    if (!genesisBlockHeader.isPresent()) {
+      LOG.error("Genesis block cannot be retriefed from chain.");
```

Comment:
```suggestion
      LOG.error("Genesis block cannot be retrieved from chain.");
```

Reply:
I don't think you actually need this check - the blockchain guarantees the genesis block is always available so you could just call `get()` directly.